borgmatic/commands/arguments.py

```diff
@@ -40,6 +43,9 @@ def parse_subparser_arguments(unparsed_arguments, subparsers):
         for subparser_name, aliases in SUBPARSER_ALIASES.items()
         for alias in aliases
     }
+    subcommand_parsers_mapping = {
+        'config': ['bootstrap'],
+    }
```

Comment:
There might be a way to introspect this information from `subparsers` instead of duplicating it here. (Do not feel strongly.)